server/scratchy_server/resources/userres.py

```diff
@@ -1,68 +1,33 @@
-import json
-from flask import Response
-from flask_restful import Resource, abort, request
-from scratchy_server.model.userModel import UserModel
-import logging
+from scratchy_server.model.userModel import UserModel, UserSchema
 from mongoengine import NotUniqueError
+from flask_apispec.views import MethodResource
+from flask_apispec import marshal_with, use_kwargs, doc
 
 
-class UserRes(Resource):
+@doc(tags=['User'])
+@marshal_with(UserSchema)
+class UserRes(MethodResource):
 
-    def get(self, userId=None):
-        if userId is None:
-            # check parameter (search roomid)
-            pseudo = request.args.get('pseudo')
```

Comment:
This feature should still by implemented and documented in the swagger (if possible). It can be done using @use_kwargs with location="query"